app/service/file_svc.py

```diff
@@ -215,6 +228,14 @@ def _get_encryptor(self):
                                    backend=default_backend())
         return Fernet(base64.urlsafe_b64encode(generated_key.derive(bytes(self.get_config('encryption_key'), 'utf-8'))))
 
+    async def _operate_extension(self, payload, headers):
+        try:
+            target = '.' + payload.split('.')[-1]
+            payload, display_name = await self.special_payloads[target](headers)
```

Comment:
shouldn't this just be `return`

Reply:
I'll set it to that, though I think that may cause a error state for the whole request (which is acceptable, considering we've already reached on by entering this space).